packages/react-hig/src/adapters/TextFieldAdapter.js

```diff
@@ -115,16 +122,18 @@ class TextFieldAdapter extends HIGElement {
           break;
         }
         default: {
-          return;
+          console.warn(
+            `${this.constructor.name} doesn't handle the prop ${propKey}`
+          );
         }
       }
     }
   }
 }
 
-const TextFieldComponent = createComponent(TextFieldAdapter);
+const TextFieldComponent = createComponent(TextField);
 
-TextFieldComponent.propTypes = {
+TextField.propTypes = {
```

Comment:
Shouldn't this reference be to the `TextFieldComponent` rather than the Text Field, since that is the React adapter we are creating?